include/rgl/api/core.h

```diff
@@ -698,3 +698,13 @@ rgl_graph_node_add_child(rgl_node_t parent, rgl_node_t child);
  */
 RGL_API rgl_status_t
 rgl_graph_node_remove_child(rgl_node_t parent, rgl_node_t child);
+
+/**
+ * Allows to set relative node's priority, which determine their execution order.
+ * This is useful when some branch (e.g. visualization data)
+ * needs to be executed before some other (e.g. publishing to ROS2).
+ * By default, all nodes have priority 0.
+ * It is illegal to set node's priority to a value lower than max priority value of its outputs.
```

Comment:
Other functions here have full documentation, with @param tags. Just for consistency, I would consider updating this dox.